airflow/www/app.py

```diff
@@ -19,8 +19,8 @@
 from flask import Flask
 from flask_admin import Admin, base
 from flask_cache import Cache
-from flask_wtf.csrf import CSRFProtect
-csrf = CSRFProtect()
+from flask_wtf.csrf import CsrfProtect
+csrf = CsrfProtect()
```

Comment:
Oops, this was a mistake. I couldn't get it to work even with reinstalls and such. I shall remove it. Nice catch.